docs/public-networks/reference/cli/options.md

```diff
@@ -724,23 +724,16 @@ Contact email address to send to the Ethstats server specified by [`--ethstats`]
     fast-sync-min-peers=8
     ```
 
-The minimum number of peers required before starting [fast synchronization](../../get-started/connect/sync-node.md#fast-synchronization).
+The minimum number of peers required before starting [fast synchronization](../../get-started/connect/sync-node.md#fast-synchronization)
+in [proof of work](../../how-to/use-pow/mining.md) networks.
 The default is 1.
 
-!!! note
+!!! important
 
-    If synchronizing in `FAST` mode, most historical world state data is unavailable. Any methods
-    attempting to access unavailable world state data return `null`.
```

Comment:
This part could be kept, and instead of says `FAST` mode, it could be changes to something like "any mode different from `FULL`"

Reply:
Does this note make sense to put here in the description of `fast-sync-min-peers` though? I'm unsure on how it's related. Would it be better under the [`sync-mode` option](https://besu.hyperledger.org/en/stable/public-networks/reference/cli/options/#sync-mode), or in the [fast sync section](https://besu.hyperledger.org/en/stable/public-networks/get-started/connect/sync-node/#fast-synchronization)?

I notice that the same note is also in [this block parameter section](https://besu.hyperledger.org/en/stable/public-networks/how-to/use-besu-api/json-rpc/?h=block+paramet#block-parameter). I'm not sure it fits well here either.

Reply:
I agree it make more sense to move it under the sync mode option